pkg/proxy/ipvs/proxier.go

```diff
@@ -1823,14 +1821,6 @@ func (proxier *Proxier) writeIptablesRules() {
 	}
 	writeLine(proxier.natRules, masqRule...)
 
-	// Install the kubernetes-specific masquerade mark rule. We use a whole chain for
-	// this so that it is easier to flush and change, for example if the mark
-	// value should ever change.
-	writeLine(proxier.natRules, []string{
-		"-A", string(KubeMarkMasqChain),
-		"-j", "MARK", "--or-mark", proxier.masqueradeMark,
```

Comment:
I don't think it's safe to delete this yet?

Reply:
Is that so? IMO, KUBE-MARK-MASQ and KUBE-MARK-DROP are similar.

Reply:
They are similar, but kube-proxy relies on kubelet to apply the drop rules in KUBE-MARK-DROP, where-as kube-proxy adds rules to KUBE-MARK-MASQ like here, which may overlap with kubelet.